assets/js/pages/Users/UserForm.jsx

```diff
@@ -52,6 +53,7 @@ function UserForm({
   const [confirmPasswordState, setConfirmPassword] = useState('');
   const [confirmPasswordErrorState, setConfirmPasswordError] = useState(null);
   const [statusState, setStatus] = useState(status);
+  const [totpState, setTotpState] = useState(totpEnabledAt !== null);
```

Comment:
Having explicitely match between a null could not be a good idea, if you put the totpEnabledAt in the props at `''` empty string by default, you could just use the boolean constructor here in useState default value `Boolean(totpEnabledAt)` this will return true if the prop is not empty